spec/models/service/retirement_management_spec.rb

```diff
@@ -102,6 +103,12 @@
     expect(@service.retirement_warn).to eq(options[:warn])
   end
 
+  it "with a Service Ansible Playbook" do
+    expect(ServiceRetireRequest).to receive(:make_request)
+      .with(nil, {:src_ids => [service_ansible_playbook.id], :__initiated_by__ => 'user', :__request_type__ => "service_retire"}, user)
+    service_ansible_playbook.class.to_s.demodulize.constantize.make_retire_request(service_ansible_playbook.id, user)
```

Comment:
Should this be `service_ansible_playbook.class.make_retire_request(service_ansible_playbook.id, user)`?